src/app/beer_garden/api/stomp/stomp_manager.py

```diff
@@ -0,0 +1,74 @@
+import logging
+from brewtils.models import Event, Events
+import beer_garden.events
+import beer_garden.router
+import threading
+from beer_garden.api.stomp.processors import EventManager
+from beer_garden.events import publish
+from beer_garden.events.processors import QueueListener
+from beer_garden.api.stomp.server import Connection
+
+logger = logging.getLogger(__name__)
+stop_thread = False
+
+
+def shutdown():
+    global stop_thread
+    stop_thread = True
+
+
+class StompManager:
```

Comment:
This is the one area I think can be simplified even more. Having a thread that runs for the life of the application and needs a way to shutdown gracefully is a pattern we reuse a lot, so much so that we already have an implementation for it.

I think this `StompManager` is a good candidate to inherit from `brewtils.stoppable_thread.StoppableThread`. Doing that should allow you to remove the `stop_thread` global.